src/app/components/Bismillah.tsx

```diff
@@ -0,0 +1,12 @@
+import { Amiri } from "next/font/google";
+
+const arabic_font = Amiri({ weight: "400", subsets: ["arabic"] })
+const Bismillah = () => {
+    return (
+        <h1 className='font-bold text-center sm:text-3xl text-2xl leading-tight' style={arabic_font.style}>
+            ﷽
+        </h1>
+    );
+};
+
+export default Bismillah;
```

Comment:
I think a good reminder to add configs for a good and consistent formatting. Endline is missing in this component.
I use Prettier so far which works great in VSCode.

Reply:
I do use prettier. I'll setup the configuration to add Endline. No worries. 👍🏻